Makefile

```diff
@@ -20,18 +20,6 @@
 all: clean build
 
 # Tag parameters
-ifndef MAJOR
-    MAJOR=2
-endif
-ifndef MINOR
-    MINOR=13
-endif
-ifndef PATCH
-    PATCH=0
-endif
-ifndef NOTES
-	NOTES=
-endif
 ifndef TAGMSG
     TAGMSG="CSI Spec 1.6"
 endif
```

Comment:
Do we really need the TAGMSG. We don't release or tag using this Makefile but rather TechOps builds. I suggest removing it unless it is also standard across repos in which case consistency may be useful.